lib/node_modules/@stdlib/blas/ext/base/sasumpw/examples/index.js

```diff
@@ -18,11 +18,12 @@
 
 'use strict';
 
-var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
-var filledarrayBy = require( '@stdlib/array/filled-by' );
-var sasumpw = require( './../lib' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var sasumpw = require( '@stdlib/blas/ext/base/sasumpw' );
```

Comment:
```suggestion
var sasumpw = require( './../lib' );
```

`examples/index.js` files should use the relative path for importing the relevant package.